website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md

```diff
@@ -1,24 +1,33 @@
 
 ---
-title: "Getting Started with eksctl"
-linkTitle: "Getting Started with eksctl"
+title: "Getting Started with Karpenter"
+linkTitle: "Getting Started with Karpenter"
 weight: 10
 description: >
-  Set up Karpenter with an eksctl cluster
+  Set up a cluster and add Karpenter 
 ---
 
-Karpenter automatically provisions new nodes in response to unschedulable
-pods. Karpenter does this by observing events within the Kubernetes cluster,
-and then sending commands to the underlying cloud provider.
+Karpenter automatically provisions new nodes in response to unschedulable pods. Karpenter does this by observing events within the Kubernetes cluster, and then sending commands to the underlying cloud provider.
 
-In this example, the cluster is running on Amazon Web Services (AWS) Elastic
-Kubernetes Service (EKS). Karpenter is designed to be cloud provider agnostic,
-but currently only supports AWS. Contributions are welcomed.
+This guide shows how to get started with Karpenter by creating a Kubernetes cluster and installing Karpenter.
+To use Karpenter, you must be running a supported Kubernetes cluster on a supported cloud provider.
+Currently, only EKS on AWS is supported.
 
-This guide should take less than 1 hour to complete, and cost less than $0.25.
+If you prefer, the following instructions use Terraform to create a cluster and add Karpenter:
+
+* [Amazon EKS Blueprints for Terraform](https://aws-ia.github.io/terraform-aws-eks-blueprints): Follow a basic [Getting Started](https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.18.0/getting-started/) guide and also add modules and add-ons. This includes a [Karpenter](https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.18.0/add-ons/karpenter/) add-on that lets you bypass the instructions in this guide for setting up Karpenter.
+
+Although not supported, you could also try Karpenter on other Kubernetes distributions running on AWS. For example:
+
+* [kOps](https://kops.sigs.k8s.io/operations/karpenter/): These instructions describe how to create a kOps Kubernetes cluster in AWS that includes Karpenter.
```

Comment:
Thoughts on moving this to the Getting started index page?

Reply:
Done.